brewblox_ctl_lib/config_commands.py

```diff
@@ -240,8 +235,18 @@ def action(self):
             '{}docker-compose pull'.format(self.optsudo),
             'sudo {} -m pip install -U brewblox-ctl'.format(PY),
             *self.lib_commands(),
+
+        ]
+
+        if confirm('Do you want to flash the firmware?'):
```

Comment:
I would change this to:
Do you want to upgrade your Spark to the latest firmware?

Some users might not know what flashing firmware means.

Reply:
Changed to 'Do you want to update your Spark controller firmware?' to preserve consistency in update vs upgrade.

Other documentation also explicitly refers to `Spark controller` or `Spark service`.